docs-2.0/2.quick-start/6.cheatsheet-for-ngql.md

```diff
@@ -452,10 +451,6 @@
   | `SHOW JOBS`          | Lists all the unexpired jobs in the current graph space.                           |
   | `STOP JOB`           | Stops jobs that are not finished in the current graph space.                               |
   | `RECOVER JOB`        | Re-executes the failed jobs in the current graph space and returns the number of recovered jobs.     |
-<!-- balance-3.1
```

Comment:
Do we need to add `submit job balance data in zone` ?

Reply:
yes, we can although there is one in **the submit job balance** section.